settings.py

```diff
@@ -25,4 +25,9 @@
 
 INSTALLED_APPS = [
     "db",
+    "django.contrib.auth",
+    "django.contrib.contenttypes",
+
 ]
+
+AUTH_USER_MODEL = "db.User"
```

Comment:
The `AUTH_USER_MODEL` is correctly set to use the custom `User` model from the `db` app. Ensure that this model is fully implemented and tested.